src/Compilers/CSharp/Portable/Symbols/ReducedExtensionMethodSymbol.cs

```diff
@@ -373,6 +373,9 @@ internal override bool IsExplicitInterfaceImplementation
             get { return false; }
         }
 
+        // extension method has no 'this' parameter
```

Comment:
Coudl be ambiguous as there is a literal use of `this` in an extension method. Consider stating that it's `static` hence can't be declared `readonly`